SConstruct

```diff
@@ -345,7 +347,7 @@ def scons():
 
     env.Alias('install', '$PREFIX')
     platform_arm = is_platform_arm()
-    Export('DAOS_VERSION', 'env', 'prereqs', 'platform_arm')
+    Export('DAOS_VERSION', 'env', 'prereqs', 'platform_arm', 'API_VERSION')
```

Comment:
I guess we also don't need the export of DAOS_VERSION unless it's used elsewhere

Reply:
`./src/control/SConscript:    Import('env', 'prereqs', 'DAOS_VERSION')`